proto/p4/p4runtime.proto

```diff
@@ -50,88 +78,56 @@ service P4Runtime {
   // Reads 'CounterEntry's.
   rpc CounterRead(CounterReadRequest) returns (stream CounterReadResponse) {
   }
```

Comment:
FYI I do not use the Counter Read/Write message. If it is OK with Antonin you can remove.